Jenkinsfile_CNP

```diff
@@ -10,69 +10,73 @@ def component = "frontend"
 
 def setupTestSecrets() {
 
-  def bootstap_env = env.ENV == "prod" || env.ENV == "demo" || env.ENV == "sbox" ? env.ENV : "stg"
+    def bootstap_env = env.ENV == "prod" || env.ENV == "demo" || env.ENV == "sbox" ? env.ENV : "stg"
```

Comment:
I know this isn't your spelling error but please make it bootstrap instead of bootstap because it's bothered me for months :p

Reply:
Corrected spelling :)